src/components/Patients/AddPatient.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react'
```

Comment:
Conventions for naming:

- `Patients.js`
- `PatientCreate.js`
- `PatientList.js`
- `PatientDetails.js`
- `PatientDelete.js`
- ...